events/schema.md

```diff
@@ -0,0 +1,33 @@
+# Events schema
+
+Events published to the bus have a standard schema that allows SDKs and services to consume them homogeneously.
+
+The schema is based on JSON (and can be modelled with JSON Schema if necessary).
+
+```
+{
+    "eventId": "448278c4-22d6-11e8-b467-0ed5f89f718b",
+    "happenedAt": "2018-03-08T12:00:00+00:00",
+    "aggregate": {
```

Comment:
The question remains as whether this can be made compact as in `"aggregate": "bot/article-run/6551f09a-bd23-4f12-a039-f8fa78ef776a"`